semantic-conventions/README.md

```diff
@@ -92,11 +92,10 @@ semantic conventions that have the tag `network`.
 
 The image supports [Jinja](https://jinja.palletsprojects.com/en/2.11.x/) templates to generate code from the models.
 
-For example, opentelemetry-java [generates typed constants for semantic conventions](https://github.com/open-telemetry/opentelemetry-java/blob/main/semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java)
```

Comment:
It seems the comment from @arminru https://github.com/open-telemetry/build-tools/pull/252/files#r1445975530 is still not fixed.

Reply:
Apologies, this is my first issue. I accepted all of their suggestions, please let me know if anything else needs to be done. Thanks